pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -69,11 +69,12 @@ type CronWorkflowSpec struct {
 	When string `json:"when,omitempty" protobuf:"bytes,12,opt,name=when"`
 }
 
-// v3.6 and after: StopStrategy defines if the CronWorkflow should stop scheduling based on a condition
+// StopStrategy defines if the CronWorkflow should stop scheduling based on an expression. v3.6 and after
 type StopStrategy struct {
-	// v3.6 and after: Condition is an expression that stops scheduling workflows when true. Use the
+	// Expression is an expression that stops scheduling workflows when true. Use the `cronworkflow`
 	// variables `failed` or `succeeded` to access the number of failed or successful child workflows.
```

Comment:
```suggestion
	// v3.6 and after: Expression is an expression that stops scheduling workflows when true. Use the
	// variables `cronworkflow`.`failed` or `cronworkflow`.`succeeded` to access the number of failed or successful child workflows.
```

I'd suggest a different workaround, leave the dot without backticks. That way it'll still copy+paste and read correctly, just have awkward formatting that is not that noticeable

Reply:
Dirty hack, I like it.